templates/content/about.html.jinja2

```diff
@@ -43,20 +43,21 @@
 
   <dl>
     <dt>Chair</dt>
-    <dd>Iris Scoffin, third year computer science undergraduate</dd>
+    <dd>Joe Krystek-Walton, third year computer science undergraduate</dd>
 
     <dt>Secretary</dt>
-    <dd>Chester Wringe, computer science research postgraduate</dd>
+    <dd>Lily Amber, first year maths undergraduate</dd>
+    <dd>Leo Millar, first year computer science undergraduate</dd>
```

Comment:
"Leo Millar, first year computer science undergraduate" to be "Leo Millar, first year computer science and maths undergraduate"